community-containers/local-ai/local-ai.json

```diff
@@ -22,22 +34,14 @@
                     "source": "nextcloud_aio_localai_images",
                     "destination": "/tmp/generated/images/",
                     "writeable": true
-                },
-                {
-                    "source": "%NEXTCLOUD_DATADIR%",
-                    "destination": "/nextcloud",
-                    "writeable": false
                 }
             ],
             "enable_nvidia_gpu": false,
             "nextcloud_exec_commands": [
-                "mkdir '/mnt/ncdata/admin/files/nextcloud-aio-local-ai'",
-                "touch '/mnt/ncdata/admin/files/nextcloud-aio-local-ai/models.yaml'",
-                "echo 'Scanning nextcloud-aio-local-ai folder for admin user...'",
-                "php /var/www/html/occ files:scan --path='/admin/files/nextcloud-aio-local-ai'",
                 "php /var/www/html/occ app:install integration_openai",
                 "php /var/www/html/occ app:enable integration_openai",
-                "php /var/www/html/occ config:app:set integration_openai url --value http://nextcloud-aio-local-ai:8080",
+                "php /var/www/html/occ config:app:set integration_openai url --value http://nextcloud-aio-local-ai:8000",
+                "php /var/www/html/occ config:app:set integration_openai api_key --value $LOCAL_AI_API_KEY",
```

Comment:
this is not going to work for now... Needs https://github.com/nextcloud/all-in-one/discussions/3889

Reply:
![image](https://github.com/user-attachments/assets/efcded6a-e7da-4f49-8f29-6e38efd09f28)